src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -587,8 +606,9 @@ const getCustomizations = (state: RootState, orgID: string): Customizations => {
     });
     files.push({
       path: '/usr/local/sbin/register-satellite',
-      data: btoa(encodeURIComponent(satCmd)),
+      data: btoa('#!/bin/bash\n' + satCmd + '\n'),
```

Comment:
adding shebang - here we kinda do not care if there is any shebang or newline. But we have properly documented that users are just supposed to add the command, so I think we are good here.